Adds XNAT data request form for first-line information collection

dangerdika opened the pull request:
Updates XNAT description and adds XNAT flow diagram of connectivity

dangerdika committed:
Adds XNAT data request form for first-line information collection

Updates XNAT description and adds XNAT flow diagram of connectivity

heyhaleema commented on _platforms/xnat.md:
```diff
@@ -5,14 +5,30 @@ title: XNAT
 
-[XNAT](https://xnat.org/) is an open source imaging informatics platform capable of storing and managing medical images and associated data. Developed by the Neuroinformatics Research Group at Washington University, XNAT can be used to support a wide range of imaging-based projects.
+[XNAT](https://xnat.org/) is an open-source imaging informatics platform for storing and managing medical 
+images and associated data for research purposes. At GSTT we used XNAT to retrieve, anonymise and manage medical imaging 
```
``` suggestion
images and associated data for research purposes. At GSTT, we use XNAT to retrieve, anonymise, and manage medical imaging 
```

heyhaleema commented on _platforms/xnat.md:
```diff
@@ -5,14 +5,30 @@ title: XNAT
+XNAT is only available to GSTT employees with access to GSTT network and before access can be granted an online teaching and 
+training session is held. If you need access to XNAT, please contact the <a href="mailto:CSCTeam@gstt.nhs.uk">CSC team</a>.
+
+A wealth of information, including our SOPs, anonymisation scripts, data retrieval and fetching scripts and similar, please visit our [XNAT GitHub repository](https://github.com/GSTT-CSC/XNAT).
```
```suggestion
For a wealth of information, including our SOPs, anonymisation scripts, data retrieval and fetching scripts and similar, please visit our [XNAT GitHub repository](https://github.com/GSTT-CSC/XNAT).
```

heyhaleema commented on _platforms/xnat.md:
```diff
@@ -5,14 +5,30 @@ title: XNAT
-- Is compliant with the [DICOM Standards Supplement 142](https://www.dicomstandard.org/News-dir/ftsup/docs/sups/sup142.pdf)
+<figure>
+<img src="https://github.com/GSTT-CSC/XNAT/blob/main/assets/anon-flowchart.jpg?raw=true" width="100%">
+<figcaption>Schematic representation of XNAT setup at GSTT.</figcaption>
```
Is there a way to make front size of the figure caption smaller (or italicised)? At first glance, it's easy to not notice the distinction between that and the body text.

![Screenshot 2023-04-11 at 10 11 52](https://user-images.githubusercontent.com/67117138/231113639-1a3357ad-8aae-4702-accd-6e5c5c62a6d8.png)

heyhaleema reviewed (changes requested):
@dangerdika Looks great 🔥 I've added a few comments/suggested changes, but they're not required per se e.g., some of them are subjective. 

Once you've resolved the conversations by either taking on the suggested changes or passing on them, I can then go ahead with re-reviewing, approving, and merging.

dangerdika committed:
Update _platforms/xnat.md

Co-authored-by: Haleema A. <67117138+heyhaleema@users.noreply.github.com>

dangerdika committed:
Update _platforms/xnat.md

Co-authored-by: Haleema A. <67117138+heyhaleema@users.noreply.github.com>

dangerdika committed:
Update _platforms/xnat.md

Co-authored-by: Haleema A. <67117138+heyhaleema@users.noreply.github.com>

dangerdika committed:
Updates sass to include figcaption css

This makes the figcaption text 80% size of the main text & changes colour to a dark grey.
